supervisor.go

```diff
@@ -0,0 +1,193 @@
+package druid
+
+import (
+	"strings"
+	"time"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	supervisorPathPrefix             = "druid/indexer/v1/supervisor"
+	supervisorSpecPathPrefix         = "druid/indexer/v1/supervisor/:supervisorId"
```

Comment:
Are this and all below really the prefixes?

How are you going to do the replacement of `:supervisorId`?

Can we user the once defined prefix for the rest of paths?

Reply:
I fixed naming `PathPrefix` -> `Endpoint`. supervisorId is applied by `applySupervisorId` function